evm/src/generation/state.rs

```diff
@@ -39,6 +39,10 @@ pub(crate) struct GenerationState<F: Field> {
     /// useful to see the actual addresses for debugging. Here we store the mapping for all known
     /// addresses.
     pub(crate) state_key_to_address: HashMap<H256, Address>,
+
+    /// Prover inputs containing the result of a MODMUL-related operation, in reverse order so that the next
```

Comment:
"reverse order" doesn't mean much. I'd write that it's little-endian order, so it becomes big-endian when `pop`ped.